.github/workflows/contract-analysis.yml

```diff
@@ -9,9 +10,8 @@ on:
     - cron: "30 5 * * *"
   workflow_dispatch:
 
-name: "Contract Security Analysis"
 jobs:
   contract_analysis:
     name: "Shared"
-    uses: aurora-is-near/.github/.github/workflows/contract_analysis.yml@master
+    uses: aurora-is-near/.github/.github/workflows/security_analysis.yml@master
```

Comment:
It looks like black box. And CI fails. It seems better to create engine proposals before such changes, as in engine we run all via `cargo make`.